packages/nuxt-locokit/src/runtime/styles/primevue/_datepicker.scss

```diff
@@ -1,25 +1,26 @@
 .p-datepicker {
-  padding: 0;
+  padding: 0.5rem;
+  background: #ffffff;
```

Comment:
Are you sure we need to fix this white color in the code ?
Don't you think it could be more maintainable to set a `--bg-color` variable set to `#fff` or something like this ?

Reply:
It is legacy from v0.7 or improvement to adapt new style
See https://github.com/locokit/locokit/issues/171